src/HotChocolate/Core/src/Subscriptions.RabbitMQ/ActiveConsumer.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using RabbitMQ.Client.Events;
+
+namespace HotChocolate.Subscriptions.RabbitMQ;
+
+public class ActiveConsumer
+{
+    public AsyncEventingBasicConsumer Consumer { get; }
+
+    public ActiveConsumer(AsyncEventingBasicConsumer consumer, Action onEmpited)
```

Comment:
do you mean emitted? onEmpited?

Reply:
It's `onEmpited`, a callback called anytime consumer's source stream (resp. listen delegates) returns to 0.

Reply:
The its OnEmptied

Reply:
it is I fixed it in commit 4e891d5